### ISSUE TITLE
Menu open/close events not working

### ISSUE THREAD

**Issue by mrbrianevans** [svelteuidev/svelteui#217]
### What package has an issue

@svelteuidev/core

### A clear and concise description of what the bug is

The `<Menu />` component events are not firing as expected.
```svelte
<Menu on:open={()=>console.log('opened')}/>
```
Doesn't log anything when the menu is opened.

### In which browser(s) did the problem occur?

Firefox

### Steps To Reproduce

Make a component like this:
```svelte
<Menu on:open={()=>console.log('opened')}/>
```
try opening the menu a few times with the browser console open.

Observe that nothing is logged.

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

### Relevant Assets

I've made a storybook story to demonstrate the problem I'm facing: 
https://github.com/mrbrianevans/svelteui/commit/7a62ea0ce353b4dc7eac87d5d3456747233ac021

**Comment by mrbrianevans** [comment]
I've looked into it briefly, and not been able to find what's causing the problem. Not sure if I'm even using the event handlers in the correct way. If anyone can point out the cause, I'm happy to contribute a fix.

**Comment by BeeMargarida** [comment]
Thank you @mrbrianevans. I'll investigate the events problem, it's probably a bug

**Comment by BeeMargarida** [comment]
Sorry, this got forgotten. I looked into it for a bit and it seems more complex that I though. For some reason the component callbacks are not being populated, so it appears like it does not have any callbacks to call on dispatch. I'll need to take some more time to identify the problem here

**Comment by BeeMargarida** [comment]
Fixed in new release 0.7.7
